src/OrbitGl/SourceCodeReport.h

```diff
@@ -39,6 +39,8 @@ class SourceCodeReport : public CodeReport {
   absl::flat_hash_map<uint32_t, uint32_t> number_of_samples_per_line_;
   uint32_t total_samples_in_function_ = 0;
   uint32_t total_samples_in_capture_ = 0;
+  uint32_t min_line_number_ = 0;
```

Comment:
Please initialize this variables here instead of inside the constructor

Reply:
changed.